tests/Functional/GetRateTest.php

```diff
@@ -21,10 +21,13 @@ public function testGetSpecificRateRecords()
         $rates = FixtureFactory::createPorter()->import(new ImportSpecification(new GetSpecificRate($this->apiKey)))
             ->findFirstCollection();
 
-        $this->assertSame('BTC', $rates->getBase());
-        $this->assertSame('USD', $rates->getQuote());
-        $this->assertLessThanOrEqual(time(), strtotime($rates->getTime()));
-        $this->assertInternalType('float', $rates->getRate());
+        $rateRecords = $rates->toAssociativeArray();
```

Comment:
This should not be necessary. Use `Porter::importOne()` instead of `import()`.

Reply:
I think you add the wrong place for this comment.
I cannot see any code that is related to the ```import()```.

Reply:
Can you give some sample code about this?
I don't understand ```Porter::importOne()``` usage clearly.

Thanks.

Reply:
`import()` returns the collection from the resource. `importOne()` does the same thing, except it returns the first record from the collection instead of the collection itself (and ensures there are no more records).